docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,120 @@
+How to create a package slice for Chisel
+****************************************
+
+If your package doesn't yet have the slice definitions you actually need to **create your own slice definition** (which you can, later on, propose to be merged upstream for everyone else to use :ref:`publishslice` ).
+
+**Let's assume you are trying to create a slice definition for installing the OpenSSL binary into your ROCK!**
+
+Make sure the slice definition doesn't exist yet
+------------------------------------------------
+
+To avoid re-creating a slice that already exists, check the following to see if something that fits your needs already exists:
+
+#. Look into the upstream `chisel-releases <https://github.com/canonical/chisel-releases/tree/ubuntu-22.04/slices>`_ repository
+#. Switch to the branch corresponding to the desired Ubuntu release for your ROCK
+#. Search your package name within the list of slice definitions files
+
+   * if you find it, open it and try to find a slice name containing the bits and pieces you need from that package
+
+
+Structure of a slice definitions file
```

Comment:
This section feels a bit disconnected from the rest of the guide: it reads more like a reference. Is it indispensable here?

Reply:
could we keep this one here, for now, and add an issue for me to move it to the Chisel reference? (I can't move it now, because we don't have a Chisel reference, and I don't think it should be in the Rockcraft documentation)

Reply:
created https://github.com/canonical/rockcraft/issues/157